### DISCUSSION TITLE
Are CVE-2021-38115 and CVE-2021-40812 fixed in 2.3.2?

### DISCUSSION THREAD

**Discussion by fdavis99** [libgd/libgd#830] [Q&A]
Please add a note to CHANGELOG with these CVEs if they are fixed:

https://nvd.nist.gov/vuln/detail/CVE-2021-38115 and https://nvd.nist.gov/vuln/detail/CVE-2021-40812 apply to libgd through 2.3.2, but there is no mention in CHANGELOG.

https://github.com/meweez claims to have fixed it in https://github.com/libgd/libgd/pull/711/commits/8b111b2b4a4842179be66db68d84dda91a246032 (patch) but that says it can't find the commit.

Links from CVE-2021-38115:
https://github.com/libgd/libgd/issues/697
https://github.com/libgd/libgd/pull/711/commits/8b111b2b4a4842179be66db68d84dda91a246032

Links from CVE-2021-40812:
https://github.com/libgd/libgd/commit/6f5136821be86e7068fcdf651ae9420b5d42e9a9
https://github.com/libgd/libgd/issues/750#issuecomment-914872385

Thanks.

**Comment by cmb69** [comment]
CVE-2021-38115 has been fixed with https://github.com/libgd/libgd/commit/8b111b2b4a4842179be66db68d84dda91a246032, which is available as of libgd 2.3.3.

CVE-2021-40812 has been *partially* fixed with https://github.com/libgd/libgd/commit/6f5136821be86e7068fcdf651ae9420b5d42e9a9, which is available as of libgd 2.3.3. The issue is not yet fixed for GIFs.
